supervisor/shared/translate.c

```diff
@@ -46,10 +47,18 @@ STATIC int put_utf8(char *buf, int u) {
     if(u <= 0x7f) {
         *buf = u;
         return 1;
+    } else if(MP_ARRAY_SIZE(ngrams) <= 64 && u <= 0xbf) {
```

Comment:
The switch from 0x80 to 0xe000 here is at 64 but above in encode and decode is 32. Shouldn't they be the same? Could we do it at compile time here with a macro?

Reply:
The Python code works based on the count of bigrams (up to 32 of them) but the C array gets 2 entries for each bigram, so 64 is correct.  However, it is true that we could place macros such as BIGRAM_CODE_POINT_START, BIGRAM_CODE_POINT_END in compression.generated.h and use them instead.

Reply:
Ah! Still plan on updating this?